trio/_core/_tests/test_run.py

```diff
@@ -2075,21 +2132,18 @@ async def bad_detach():
         nursery.start_soon(bad_detach)
 
 
-async def test_detach_and_reattach_coroutine_object():
-    unrelated_task = None
-    task = None
+async def test_detach_and_reattach_coroutine_object() -> None:
+    tasks: dict[str, _core.Task] = {}
```

Comment:
Why did you not use `not_none` here when you did so for similar situations elsewhere?

Reply:
I must have edited this one earlier, before I swapped to `not_none()`. Will switch this one to that too.